tests/functional/test_install_reqs.py

```diff
@@ -191,6 +191,42 @@ def test_install_local_editable_with_subdirectory(script):
     result.assert_installed('version-subpkg', sub_dir='version_subdir')
 
 
+def test_user_with_prefix_in_pydistutils_cfg(script, data, virtualenv):
+    """
+    Test that using --home works when pydistutils.cfg configures a prefix
+    """
+
```

Comment:
This blank line however definitely not appropriate - PEP8 advises 'sparing use' of vertical whitespace. This breaks up the code so it looks like the body is separate to the definition.